### PR TITLE
docs: grammar and improvements

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

**Comment by Rexogamer** [comment]
Build failures are unrelated to my changes as I've only touched the docs.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Search Engine Parser
 
-<span><i>"If it is a search engine, then it can be parsed"</i> - Some random guy</span>
+<span><i>"If it is a search engine, then it can be parsed"</i> - some random guy</span>
 
 ![Demo](assets/animate.gif)
 
 [![Python 3.5|3.6|3.7|3.8](https://img.shields.io/badge/python-3.5%7C3.6%7C3.7%7C3.8-blue)](https://www.python.org/downloads/)
-[![PyPI version](https://badge.fury.io/py/search-engine-parser.png)](https://badge.fury.io/py/search-engine-parser)
```
Comment (MeNsaaH) [inline README.md:8]: Why change the source of the badge?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+![PyPI version](https://img.shields.io/pypi/v/search-engine-parser)
 ![PyPI - Downloads](https://img.shields.io/pypi/dm/search-engine-parser)
 [![Build Status](https://travis-ci.com/bisoncorps/search-engine-parser.svg?branch=master)](https://travis-ci.com/bisoncorps/search-engine-parser)
 [![Documentation Status](https://readthedocs.org/projects/search-engine-parser/badge/?version=latest)](https://search-engine-parser.readthedocs.io/en/latest/?badge=latest)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors)
```
Comment (MeNsaaH) [inline README.md:13]: Why change the style?

**Review by MeNsaaH — COMMENTED** [review]
Everything looks good. I don't really know why you are changing the badge type and providers. Is there any reason?

File: README.md (hunk 2 above)
Comment (MeNsaaH) [reply README.md:13]: This makes sense. To make it uniform with the rest, nicee

Comment (Rexogamer) [reply README.md:13]: It was inconsistent with the other badges.

File: README.md (hunk 1 above)
Comment (Rexogamer) [reply README.md:8]: Old one was pretty blurry, also just fits better

Comment (MeNsaaH) [reply README.md:8]: Alright

**Pull request merged**